elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -168,6 +168,8 @@ elytron.syslog-audit-log.transport=The transport to use to connect to the syslog
 elytron.syslog-audit-log.format=The format to use to record the audit event.
 elytron.syslog-audit-log.host-name=The host name to embed withing all events sent to the remote syslog server.
 elytron.syslog-audit-log.ssl-context=The SSLContext to use to connect to the syslog server when SSL_TCP transport is used.
+elytron.syslog-audit-log.syslog-format=The RFC format to be used for describing the audit event.
```

Comment:
It might be good to indicate the default value here.

Reply:
Fixed

Reply:
I originally updated it to reference RFC5424 but removed it based on https://github.com/wildfly/wildfly-core/pull/3856#pullrequestreview-276419748 . I could add the section detailing the possible values?